resources/clansettings.json

```diff
@@ -119,6 +119,43 @@
             false
         ]
     },
+    "freshkill_tactics":{
+        "status wise":[
+            "status wise",
+            "The Clan will be fed according their status and within the status with age.",
+            true
+        ],
+        "younger first":[
+            "younger first",
+            "The Clan will be fed according to their age only.",
+            false
+        ],
+        "less nutrition first":[
+            "less nutrition first",
+            "First the cats with the lowest nutrition will be fed, afterwards the Clan will be fed according to their status.",
+            false
+        ],
+        "hunter first":[
+            "hunter first",
+            "Firstly the hunter will be fed, afterwards the Clan will be fed according to their status.",
```

Comment:
From this explanation, I would assume that cats able to hunt (warriors and warrior apprentices) would be fed first.  However, this tactic works by feeding cats with the "HUNTER" skill first, than everyone else. I suggest rewording the hover-text blurb.